code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -841,6 +841,11 @@
 	reagents.add_reagent(NUTRIMENT, 3)
 	bitesize = 2
 
+/obj/item/weapon/reagent_containers/food/snacks/faggot/processed/New()
+	..()
+	reagents.clear_reagents()
+	bitesize = 2
```

Comment:
Rereading this, I noticed there's really no reason to set this again, right? It's already set to 2 on the objects themselves?

Reply:
Will fix later today